.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: python
+python: "2.7"
+
+# Use the new container infrastructure
+sudo: false
+
+# Install ansible
+addons:
+  apt:
+    packages:
+    - python-pip
+
+install:
+  # Install ansible
+  - pip install ansible
```

Comment:
Use a specific version of ansible here. There have been too many breaking changes recently to expect them to follow semantic versioning.